internal/action/reminder.go

```diff
@@ -0,0 +1,34 @@
+package action
+
+import (
+	"context"
+	"os"
+
+	"github.com/gopasspw/gopass/internal/out"
+	"github.com/gopasspw/gopass/pkg/ctxutil"
+)
+
+func (s *Action) printReminder(ctx context.Context) {
+	if !ctxutil.IsInteractive(ctx) {
+		return
+	}
+	if !ctxutil.IsTerminal(ctx) {
+		return
+	}
+	if sv := os.Getenv("GOPASS_NO_REMINDER"); sv != "" {
+		return
+	}
+
+	if s.rem.Overdue("fsck") {
+		out.Notice(ctx, "You haven't run 'gopass fsck' in a while.")
+		return
```

Comment:
Note that a return statement here means that we will warn only about fsck even if we should also warn about updates and audit.

People might ignore `fsck` but be more interested in the `update` one I guess.

Reply:
Actually this is intentional! I did only want to print one notice per day.
Added a comment and put update first.